lib/astra-notices/composer.json

```diff
@@ -2,23 +2,32 @@
     "name": "brainstormforce/astra-notices",
     "type": "wordpress-plugin",
     "description": "Easily create admin notices",
-    "version": "1.1.8",
-    "license": "GPL v3",
+    "license": "GPL-3.0-or-later",
```

Comment:
keep this as earlier "license": "GPL v3",

Reply:
reverted changes

Reply:
revert this change - "license": "GPL-3.0-or-later",